client/src/interface/NotificationApData.ts

```diff
@@ -0,0 +1,10 @@
+import { Notification } from "./Notification";
```

Comment:
just a heads up, I think the name of the file is misspelled `NotificationApiData.ts` rather than `NotificationApData.ts`

Reply:
Oh right! Thank you!